src/components/tables/EntityTable.tsx

```diff
@@ -186,10 +202,10 @@ function EntityTable({
         filterTable: debounce(
             (event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
                 const filterQuery = event.target.value;
+                const hasQuery = Boolean(filterQuery && filterQuery.length > 0);
+                isFiltering.current = hasQuery;
```

Comment:
Super easy way to decide if the table is empty because data does not exist or because someone is filtering